Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -2619,6 +2619,33 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     fireCommandEventIfNeeded(command, activeTab.get());
 }
 
+#if TARGET_OS_IPHONE
+WebExtensionCommand* WebExtensionContext::command(UIKeyCommand *keyCommand)
```

Comment:
This name could be a little bit more descriptive. `commandForKeyCommand`? `matchingCommand`?

Reply:
Yeah I think so too. 

We do have these 2 in the same class though:
```
    WebExtensionCommand* command(const String& identifier);
    WebExtensionCommand* command(NSEvent *);
```
Hence me naming it the same.

I'll call it commandMatchingKeyCommand()